runscan.sh

```diff
@@ -10,14 +10,18 @@ USER=$(whoami)
 
 # fetch the latest version of the chosen branch of Privacy Badger
 if [ -e "$PB_DIR" ] ; then
-  echo "Updating Privacy Badger..."
+  echo "Updating Privacy Badger on branch: $PB_BRANCH"
```

Comment:
The output of runscan.sh does not end up in log.txt.

Reply:
Also, you don't need runscan.sh at all to run a scan; you could use crawler.py directly. So all the pertinent info should be logged by crawler.py